test/openAPI/features/wallet_linked_authenticate.feature

```diff
@@ -0,0 +1,47 @@
+    And The /linked-authorization/authenticate endpoint response should have content-type: application/json header
+    And The /linked-authorization/authenticate endpoint response should contain linkedTransactionId
+    And The /linked-authorization/authenticate endpoint response should match json schema without errors
+
+  @negative
+  Scenario: Not able to check the correctness of the data because of an invalid linkedTransactionId
+    Given Wants to check the correctness of the data
+    When Send POST /linked-authorization/authenticate request with given invalid linkedTransactionId
+    Then Receive a response from the /linked-authorization/authenticate endpoint
+    And The /linked-authorization/authenticate endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/authenticate endpoint response should have status 200
+    And The /linked-authorization/authenticate endpoint response should have content-type: application/json header
+    And The /linked-authorization/authenticate endpoint response should match json schema
+    And The /linked-authorization/authenticate response should contain errorCode property equals to "invalid_transaction_id"
+
+  @negative
+  Scenario: Not able to check the correctness of the data because of an invalid individualId
+    Given Wants to check the correctness of the data
+    When Send POST /linked-authorization/authenticate request with given invalid individualId
+    Then Receive a response from the /linked-authorization/authenticate endpoint
+    And The /linked-authorization/authenticate endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/authenticate endpoint response should have status 200
+    And The /linked-authorization/authenticate endpoint response should have content-type: application/json header
+    And The /linked-authorization/authenticate endpoint response should match json schema
+    And The /linked-authorization/authenticate response should contain errorCode property equals to "invalid_identifier"
+
+  @negative
+  Scenario: Not able to check the correctness of the data because of an invalid challengeList
+    Given Wants to check the correctness of the data
+    When Send POST /linked-authorization/authenticate request with given invalid challengeList
+    Then Receive a response from the /linked-authorization/authenticate endpoint
+    And The /linked-authorization/authenticate endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/authenticate endpoint response should have status 200
+    And The /linked-authorization/authenticate endpoint response should have content-type: application/json header
+    And The /linked-authorization/authenticate endpoint response should match json schema
+    And The /linked-authorization/authenticate response should contain errorCode property equals to "invalid_no_of_challenges"
```

Comment:
can we add one test case with invalid challenge and expect auth failed ?

Reply:
Sure! Which challenges supposed to be invalid ?

Reply:
we can try with PIN challenge, where we can take from configuration a invalid PIN and expect auth failed

Reply:
Had a new comment in the new PR (https://github.com/GovStackWorkingGroup/bb-identity/pull/47#discussion_r1241378478), so marking this as resolved.